src/pages/core/entrypoints.mdx

```diff
@@ -128,7 +127,6 @@ struct CustomInstantiateMsg {
   PartialEq,
   schemars::JsonSchema
 )]
-#[serde(deny_unknown_fields)]
```

Comment:
This was removed from `#[cw_serde]` (see CosmWasm changelog).

I am wondering about `#[serde(rename_all = "snake_case")]` but that seems to be enum specific. Maybe we should move the whole explanation of `#[cw_serde]` to a dedicated page?

Reply:
Yeah, we should probably move this out. I think I wrote those docs before the attribute was removed with 2.1.

And yes, the snake case rename is enum specific.